sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -68,10 +68,12 @@
   private static final String LATEST_KEYWORD = "LATEST";
   private Mode mode;
   private boolean ignoreError;
+  private boolean includeArchives;
 
-  public UpdateCenterDeserializer(Mode mode, boolean ignoreError) {
+  public UpdateCenterDeserializer(Mode mode, boolean ignoreError, boolean includeArchives) {
     this.mode = mode;
-    this.ignoreError = ignoreError;
+    this.ignoreError = ignoreError; // || includeArchives;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)